src/crimson/os/seastore/seastore_types.h

```diff
@@ -722,6 +722,184 @@ struct blk_paddr_t : public paddr_t {
   }
 };
 
+template <typename T>
+class block_map_t {
```

Comment:
This is only used by RBMSpaceTracker, can we move this over the async_cleaner.h?